notebook/services/api/api.yaml

```diff
@@ -654,12 +639,18 @@ definitions:
       id:
         type: string
         format: uuid
-      notebook:
-        type: object
-        properties:
-          path:
-            type: string
-            description: path to notebook
+      path:
+        type: string
+        description: "The path to the session.  For console sessions, the path 
+         contains the cwd of the kernel and the name of the console session, 
+         which may be a uuid"
+      type:
+        type: string;
+        description: session type
+        enum:
+          - notebook
+          - file
+          - console
```

Comment:
I have mixed feelings about CWD, since it bakes filesystem knowledge into the API. When interpreting cwd, is the CM path used, or the real filesystem? We want a kernel to start in a particular dir when the notebook it starts with is on the same filesystem, but it's unclear what should happen when the Contents paths don't map onto the kernel's filesystem, or how/when that should be communicated.